### PR TITLE
feat: add copy method for logical_operator

### PR DESCRIPTION
**Pull request by stephaniewang526**

### CODE REVIEW

File: src/include/duckdb/planner/logical_operator.hpp [hunk 1]
```diff
@@ -77,6 +77,8 @@ class LogicalOperator {
 
 	static unique_ptr<LogicalOperator> Deserialize(Deserializer &deserializer, PlanDeserializationState &state);
 
+	static unique_ptr<LogicalOperator> Copy(unique_ptr<LogicalOperator> &op, ClientContext &context);
```
Comment (Mytherin) [inline src/include/duckdb/planner/logical_operator.hpp:80]: Could we define this to be more consistent with all the other Copy functions in the system? i.e.:

```sql
virtual unique_ptr<LogicalOperator> Copy() const = 0;
```

**Review by Mytherin — COMMENTED** [review]
Thanks for the PR! LGTM. One comment:

File: src/include/duckdb/planner/logical_operator.hpp (hunk 1 above)
Comment (stephaniewang526) [reply src/include/duckdb/planner/logical_operator.hpp]: Done. Thanks!

**Comment by Mytherin** [comment]
Thanks!

**Pull request merged**